### DISCUSSION TITLE
Switch between Assistants API Beta v1 and v2

### DISCUSSION THREAD

**Discussion by tonychoo** [openai/openai-dotnet#100] [Q&A]
Is there a way to switch between Assistants API Beta version 1 and 2 in the .NET SDK? Additionally, does the current OpenAI .NET SDK support Assistants API Beta version 1 or 2 by default?

SDK: OpenAI 2.0.0-beta.5

**Comment by janaka** [comment]
@tonychoo might be worth being more precise with your question given there are several moving parts. Maybe update with the exact package name and version strings.

**Reply by tonychoo** [reply]
Thank you, I have added the package and version info.

**Reply by janaka** [reply]
Thanks for adding the version number. But your question is still unclear to me i'm afraid. When you say "API Beta version 1 and 2" what are you exactly referring to which API? What do you mean by "switching" at runtime or at design time? Maybe details around the problem you are trying to solve would help explain what you are asking.

**Reply by tonychoo** [reply]
Hi Janaka,

I'm referring to Assistants API Beta, there are 2 versions of the api v1 legacy and v2. https://platform.openai.com/docs/api-reference/assistants-v1

What I'm not sure is if I can change the .net config or parameters to call Assistants v1 legacy api?
